pkg/config/config_test.go

```diff
@@ -16,13 +16,13 @@ import (
 )
 
 func TestConfigWithValueOverrides(t *testing.T) {
-	overrideRepo := "overrideRepo"
+	overrideRepo := "/overrideRepo"
 	overrideName := "puuid"
 	overrideClientAddress := "1.1.1.1"
 	overrideClientPort := 1234
 
 	defaultConfig := types.Bacalhau{
-		DataDir: "defaultRepo",
+		DataDir: "/defaultRepo",
```

Comment:
if relative paths are no longer supported, you should still keep a test case and validate if returns a well formated error